contents/stats.md

```diff
@@ -0,0 +1,430 @@
+This demonstrates that the **median is robust to outliers**.
+In both cases, the mode is used only for comparison, since it is not advised for use with numerical data.
+
+### Advice on Central Tendencies {#sec:stats_central_advice}
+
+You might be wondering: "which central tendency shall I use? Mean? Median? Mode?".
+Here is our advice:
+
+- For data that do *not* have outliers, use the **mean**
+- For data that *do* have outliers, use the **median**
+- For categorical/nominal data, use the **mode**
+
+## Dispersion Measures {#sec:stats_dispersion}
+
+In statistics, **dispersion is a measure of how much spread certain observations are from a central tendency**.
+It is also called variability or spread.
+One interesting property is that, contrary to central tendencies, dispersion measures can *only* be **positive**.
+In other words, we do not have negative measures of dispersion.
+
+### Variance and Standard Deviation {#sec:stats_dispersion_varstd}
+
+The first dispersion measure that we will cover are variance and standard deviation.
+We will be covering both measures together because they have a profound relationship.
+**The variance is the square of the standard deviation and the standard deviation is the square root of the variance.**
+
+Let's start with the variance. **Variance is mean of the squared difference between measurements and their average value**:
+
+$$ \operatorname{Var}(x) = \frac{1}{n-1} \sum^n_{i=1} (x_i - \bar{x})^2. $$ {#eq:variance}
+
+We use the operator $\operatorname{Var}$ to denote variance, but you also might find variance being denoted as the squared Greek letter $\sigma^2$.
+Note that we are using $n-1$ in @eq:variance.
+This is because we need a bias correction since we are using one *degree of freedom* from our estimate mean $\bar{x}$.
+Degrees of freedom are not in the scope of our book, so we won't cover in details, but feel free to check the [Wikipedia](https://en.wikipedia.org/wiki/Degrees_of_freedom_(statistics)) for a in depth explanation.
+
+Since we are squaring the differences in @eq:variance, the variance has a property that all dispersion measures have: the variance *cannot* be negative.
+
+The **var**iance can be used from the `var` function from Julia's standard library `Statistics` module:
+
+```julia
+using Statistics: var
+```
+
+Like before, we can apply the variance to different groups in our `more_grades` `DataFrame`:
+
+```jl
+# variance
+s = """
+    gdf = groupby(more_grades(), :name)
+    combine(gdf, :grade => var)
+    """
+sco(s; process=without_caption_label)
+```
+
+We can see that Sally has the highest dispersion in her grades measured by its variance.
+
+**The squared deviation is the square root of the mean of the squared difference between measurements and their average value**.
+Or in more simple words: it is the **square root of the variance**:
+
+$$ \sigma(x) = \sqrt{\frac{1}{n-1} \sum^n_{i=1} (x_i - \bar{x})^2}. $$ {#eq:std}
```

Comment:
Ok. See, using $\sigma$ feels more natural, probably is better to do the same for variance $\sigma^2$ in the previous paragraph.